jni/CMakeLists.txt

```diff
@@ -85,15 +85,15 @@ elseif(${CMAKE_SYSTEM_PROCESSOR} STREQUAL x86_64)
     set(MACH_ARCH x64)
 endif()
 
-set(KNN_MAINTAINER "OpenDistro for Elasticsearch Team <opendistro@amazon.com>")
+set(KNN_MAINTAINER "OpenSearch Team <opendistro@amazon.com>")
```

Comment:
Will links/emails be updated later?

Reply:
Yes, thats correct. Even this is not mandatory, since it was called out during the discussion, i update it .